features/product/viewing_products/viewing_product_from_parent_level_taxon_sorted_by_position.feature

```diff
@@ -0,0 +1,28 @@
+@viewing_products
```

Comment:
I would be for merging this feature file with `viewing_product_from_specific_taxon_sorted_by_position.feature` and check sorted products by browsing two different taxons (parent and children)